src/Entity/AdherentMessage/Filter/MunicipalChiefFilter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace AppBundle\Entity\AdherentMessage\Filter;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class MunicipalChiefFilter extends AbstractAdherentMessageFilter
+{
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(nullable=true)
+     */
+    private $firstName;
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(nullable=true)
+     */
+    private $lastName;
+
+    /**
+     * @var array|null
```

Comment:
`@var array|string[]|null`

also, do you prefer `null` over an empty array ?

Reply:
it should not be nullable, fixed, thanks